packages/sdks/snippets/hydrogen/app/components/CustomHero.tsx

```diff
@@ -0,0 +1,27 @@
+// app/components/CustomHero.tsx
+import React from 'react';
+import type {RegisteredComponent} from '@builder.io/sdk-react';
+
+export function CustomHero({children}: {children?: React.ReactNode}) {
```

Comment:
React has a helper type for this called `PropsWithChildren`:
```suggestion
import React, {  type PropsWithChildren } from 'react';
import type {RegisteredComponent} from '@builder.io/sdk-react';

export function CustomHero({children}: PropsWithChildren) {
```